src/sentry/models/group.py

```diff
@@ -624,6 +624,10 @@ class Meta:
             models.Index(fields=("status", "substatus", "id")),  # TODO: Remove this
             models.Index(fields=("status", "substatus", "first_seen")),
             models.Index(fields=("project", "status", "priority", "last_seen", "id")),
+            models.Index(
```

Comment:
do we want status on these as well? since folks are probably also going to query for unresolved issues